### ISSUE TITLE
Add validations for fields

### ISSUE THREAD

**Issue by ppatriotis** [prolificinteractive/mabi#24]
We should have annotations for simple validations.

Something like @field-validation required and @field-validation read-only so that it would get rid of some common code on updates

**Comment by conord33** [comment]
Yeah this seems like a good idea.  So we would add those types of tags to the fields in the models?
